### PR TITLE
Add custom statuses notice

### PR DESCRIPTION
**Pull request by esethna**
add notice and image to release branch

### CODE REVIEW

**Comment by esethna** [comment]
@ogi-m let's please test as applicable to ensure:

1. It will appear for ALL cloud instances on or after March 11 (ie server version condition is ignored for cloud)
2. It will appear for on-prem instances on or after March 11 only if on v5.33 or later

**Comment by esethna** [comment]
cc// @amyblais this should be merged before we ship the next cloud release (March 10) assuming we enable the feature flag

File: notices.json [hunk 1]
```diff
@@ -54,5 +54,23 @@
...
+    "conditions": {
+      "audience": "all",
+      "clientType": "all",
+      "serverVersion": [">5.32"],
+      "displayDate": ">= 2021-03-11T00:00:00Z"
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Introducing custom statuses",
+        "description": "Setting a custom status adds a descriptive status message and emoji that’s visible to everyone throughout the app, allowing you to express your status in any way you prefer.",
```
Comment (cwarnermm) [inline notices.json:69]:
```suggestion
        "description": "Setting a custom status lets you express your status in any way you prefer by adding a descriptive status message and emoji that’s visible to everyone throughout the app.",
```

**Review by cwarnermm — COMMENTED** [review]
@esethna - Text update proposed inline.

**Comment by esethna** [comment]
@amyblais @ogi-m let's please merge this before the March 10 cloud release

**Comment by esethna** [comment]
@ogi-m @amyblais gentle ping on this

**Review by ogi-m — APPROVED** [review]
Tested and approved 👍🏻

**Review by cwarnermm — APPROVED** [review]
Looks good!

**Pull request merged**